certified-connectors/LAWLIFT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,660 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LAWLIFT",
+    "description": "LAWLIFT is a company focused on document automation. The editor, which has become known for its ease of use, has won several awards. Customers are large and medium-sized companies, numerous big law firms, legal departments and notary offices. For more information visit : www.lawlift.com .",
+    "version": "V1",
```

Comment:
@makaveli1313 Can you please keep the version like this as we can keep sub versions as well like this:

`"version": "1.0.0"`